website/docs/guides/orchestration/custom-cicd-pipelines/3-dbt-cloud-job-on-merge.md

```diff
@@ -58,6 +58,26 @@ Here’s a video showing these steps:
 
 <WistiaVideo id="u7mo30puql" />
 </TabItem>
+<TabItem value="ADO">
+
+In Azure:
+
+- Open up your Azure DevOps project where you want to run the pipeline (the same one that houses your dbt project)
+- Click on *Pipelines* > *Create Pipeline*
+- Select where your git code is located (should be *Azure Repos Git*)
+- Select your git repository from the list
+- Select *Starter pipeline* (this will be updated later in Step 4)
+- Click on *Variables* > *New variable*
+- In the *Name* field, input `DBT_API_KEY`
+    - **It’s very important that you copy/paste this name exactly because it’s used in the scripts below.**
+- In the *Value* section, paste in the key you copied from dbt Cloud
+- Make sure the check box next to *Keep this value secret* is checked. This will mask the value in logs, and you won't be able to see the value for the variable in the UI.
+- Click *OK* > *Save* to save the variable
+- Save your new Azure pipeline
+
+    ![View of the Azure pipelines window for entering DBT_API_KEY](/img/guides/orchestration/custom-cicd-pipelines/dbt-api-key-azure.png)
```

Comment:
adding images to `https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/static/img/guides/orchestration/custom-cicd-pipelines` and will reference using `Lightbox src=/img/guides/orchestration/custom-cicd-pipelines/image_name.jpg`